tchannel/tornado/connection.py

```diff
@@ -664,7 +663,7 @@ def keep_reading(f):
                 lambda f: io_loop.spawn_callback(self.fill),
             )
 
-        io_loop.add_future(read_message(self.io_stream), keep_reading)
```

Comment:
Agreed. I'm okay with this change mostly because it makes those log messages go away and it won't behave any differently in other cases.